### PR TITLE
Update collection creation documentation

### PR DESCRIPTION
**Pull request by Ndpnt**

### CODE REVIEW

**Review by MattiSG — APPROVED** [review]
Does this permission set also allow for releases publishing? 🙂

**Comment by Ndpnt** [comment]
> Does this permission set also allow for releases publishing? 🙂

Yep!

File: content/collections/create.en.md
Comment (clementbiron) [inline content/collections/create.en.md:167]: An example is always welcome.

File: content/collections/create.en.md
Comment (clementbiron) [inline content/collections/create.en.md:173]:
```suggestion
- Create the `SERVER_FINGERPRINT` [secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository) by using the previously ed25519 fingerprint generated
```

File: content/collections/create.en.md
Comment (clementbiron) [inline content/collections/create.en.md:202]:
```suggestion
- Save, commit and push
```

File: content/collections/create.en.md
Comment (clementbiron) [inline content/collections/create.en.md:227]:
```suggestion
- Save, commit and push
```

File: content/collections/create.en.md
Comment (clementbiron) [inline content/collections/create.en.md:245]:
```suggestion
- Save, commit and push
```

File: content/collections/create.en.md
Comment (clementbiron) [inline content/collections/create.en.md:290]:
```suggestion
- Save, commit and push
```

File: content/collections/create.en.md
Comment (clementbiron) [inline content/collections/create.en.md:289]:
```suggestion
- Attach public `<collection_name>-key.pub` and private `<collection_name>-key` keys file to the entry
```

File: content/collections/create.en.md
Comment (clementbiron) [inline content/collections/create.en.md:319]:
```suggestion
- Save, commit and push
```

**Review by clementbiron — APPROVED** [review]
I've given you some feedback on possible areas for improvement, nothing blocking.

Shouldn't we add a paragraph at the end to explain that you need to open a PR with the inventory update and that once it's merged you can check that everything's fine in the Github actions?

**Comment by Ndpnt** [comment]
> Shouldn't we add a paragraph at the end to explain that you need to open a PR with the inventory update and that once it's merged you can check that everything's fine in the Github actions?

I don't think it's necessary, but if you want to add it, feel free :)

File: content/collections/create.en.md
Comment (MattiSG) [inline content/collections/create.en.md:164]:
```suggestion
- Update the following entries in the inventory file `deployment/inventory.yml`:
```

File: content/collections/create.en.md
Comment (MattiSG) [inline content/collections/create.en.md:171]:
```suggestion
- Log in on GitHub with a user account that has admin privileges for the `<collection_name>-declarations` repository
```

File: content/collections/create.en.md
Comment (MattiSG) [inline content/collections/create.en.md:177]:
```suggestion
This key will enable automated deployment via GitHub Actions.
```

File: content/collections/create.en.md
Comment (MattiSG) [inline content/collections/create.en.md:184]: Why do we need to add the server's private key  to the server's agent? 🤔

File: content/collections/create.en.md
Comment (MattiSG) [inline content/collections/create.en.md:186]: Is that true?

File: content/collections/create.en.md
Comment (MattiSG) [inline content/collections/create.en.md:194]:
```suggestion
#### Back key up
```

File: content/collections/create.en.md
Comment (MattiSG) [inline content/collections/create.en.md:196]: Should not be a bullet point if not an action 🙂

File: content/collections/create.en.md
Comment (MattiSG) [inline content/collections/create.en.md:197]: I would assume this is known since the agent is necessarily a core team member if they have access to the shared passwords :)

File: content/collections/create.en.md
Comment (MattiSG) [inline content/collections/create.en.md:212]:
```suggestion
- Set the expiry date to the maximum possible (one year later at the time of writing)
```

File: content/collections/create.en.md
Comment (MattiSG) [inline content/collections/create.en.md:213]:
```suggestion
- In “Resource owner”, select the “OpenTermsArchive“ organization
```

File: content/collections/create.en.md
Comment (MattiSG) [inline content/collections/create.en.md:244]:
```suggestion
- Generate a new password that does not contain any quote or backtick
```

File: content/collections/create.en.md
Comment (MattiSG) [inline content/collections/create.en.md:267]:
```suggestion
- Update token in the inventory file `deployment/inventory.yml`: `ota_engine_github_token: !vault | <encrypted GitHub Token>`
```

File: content/collections/create.en.md
Comment (MattiSG) [inline content/collections/create.en.md:273]: > if long options exist, we should use them :)

File: content/collections/create.en.md
Comment (MattiSG) [inline content/collections/create.en.md:283]:
```suggestion
#### Back key up
```

File: content/collections/create.en.md
Comment (MattiSG) [inline content/collections/create.en.md:300]:
```suggestion
### Enable sending error reports over email
```

**Review by MattiSG — APPROVED** [review]
Congratulations for pulling through this not exciting but important task! I am sure we will be thankful for the work done for many months 🙇

File: content/collections/create.en.md
Comment (Ndpnt) [reply content/collections/create.en.md:184]: Great question, it seems completely useless…

File: content/collections/create.en.md
Comment (Ndpnt) [reply content/collections/create.en.md:186]: I don't think so, I'd say only `cat ~/.ssh/ota-deploy.pub >> ~/.ssh/authorized_keys`

File: content/collections/create.en.md
Comment (Ndpnt) [reply content/collections/create.en.md:197]: So, you suggest to remove it?

**Pull request merged**
